keps/sig-network/20191220-apiserver-url-envvar.md

```diff
@@ -0,0 +1,294 @@
+---
+title: Apiserver discovery URL env variable
+authors:
+  - "@anguslees"
+owning-sig: sig-network
+participating-sigs:
+  - sig-apimachinery
+  - sig-architecture
+reviewers:
+  - "@thockin"
+  - "@liggitt"
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2019-12-20
+last-updated: 2019-12-20
+status: provisional
```

Comment:
We can let this gather some feedback, but I think it's effectively implementable